### DISCUSSION TITLE
Title, subject, description: best/correct usage

### DISCUSSION THREAD

**Discussion by dror3go** [photoprism/photoprism#1132] [General]
Hi,

Is there a conceptual difference between a photo's title, subject, and description? When should I use each? Is there right and wrong here?

Thanks a lot

**Comment by lastzero** [comment]
Title, Subject, and Description are Dublin Core standard fields, see https://guides.library.ucsc.edu/c.php?g=618773&p=4306386. Note that Exif metadata doesn't contain an "official" title field (XMP does), so the description field is often used for this.

The title should be a single line and may be used as file name when downloading albums as ZIP. It is also shown in the list and cards view. Descriptions are never used a file name and only shown in the cards view underneath the title. Subject may list related people or objects so that you can find them, but it's not directly visible in search results.

**Reply by dror3go** [reply]
I also noticed "Notes" after posting the question.

@lastzero I understand what you mean, but your answer comes mostly from the implementation side of things, not much as from an end user's side of things. My dad shouldn't care about Dublin Core standards, and from the link you posted it's also not clear ("A name given to the resource", "An account of the resource").

My point is that an end user should know what's appropriate for each of the fields, and maybe how the software will treat differently each of them: where each is shown in the UI is great of course, but maybe there are other things to consider (search for example)?

I'll give you an example where I'm coming from: I scanned my family's old physical albums (taken in film back in the 80s and so), and I'm importing them to PhotoPrism.
In the physical albums, some of the photos had a text written next to them. Where would you put this text in PhotoPrism?

**Reply by lastzero** [reply]
The short answer is that there are multiple metadata standards.  Every photographer and vendor is using these fields at least slightly different. Notes is an "internal" field for information which should NOT be visible to regular users, e.g. when browsing an album. If unsure, use title and description only.

**Comment by luposlip** [comment]
**UPDATE**: My PhotoPrism server is working in the background, the logs says it reads JSON files, but I didn't realize it's actually loading in the descriptions from them now! 😃 I'm keeping this comment, in case someone else have this question in the future.

~Hey @lastzero "Essentials" subscriber here. I have just imported my 60k photo library from Google Photos. This summer I spent approximately 20 hours, writing detailed descriptions of all photos from a vacation. Unfortunately for me, they don't show up in my PhotoPrism library. Do you know how I can make that happen? I have checked the `.json` files imported into the `originals` folder, and I'm happy to see they contain the descriptions.~

~Any help is highly appreciated,~ thanks for the great work! 🥇
